src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDiagnosticsEndpoint.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
```

Comment:
Personally I've taken to doing every new class I create as `#nullable enable` to make an eventual switch easier. I find that I like the patterns it forces me into anyway, but I don't think we're making that a requirement.